ams/ams-server/src/main/java/com/netease/arctic/ams/server/utils/JacksonUtils.java

```diff
@@ -0,0 +1,133 @@
+            return INDENT_OBJECT_MAPPER.writeValueAsString(object);
+        } catch (JsonProcessingException e) {
+            throw new IllegalStateException(SERIALIZE_ERROR, e);
+        }
+    }
+
+    public static String toJSONString(Object object){
+        try{
+            return OBJECT_MAPPER.writeValueAsString(object);
+        } catch (JsonProcessingException e){
+            throw new IllegalStateException(SERIALIZE_ERROR, e);
+        }
+    }
+
+    public static <T> T parseObject(String jsonString, Class<T> clazz){
+        try {
+            return OBJECT_MAPPER.readValue(jsonString, clazz);
+        } catch (IOException e){
+            throw new IllegalStateException(DESERIALIZE_ERROR, e);
+        }
+    }
+
+    public static <T> T parseObject(String jsonString, TypeReference<T> typeReference){
+        try {
+            return OBJECT_MAPPER.readValue(jsonString, typeReference);
+        } catch (JsonProcessingException e) {
+            throw new IllegalStateException(DESERIALIZE_ERROR, e);
+        }
+    }
+
+    public static <T> T parseObjects(String jsonString, Class<T> clazz){
+        JavaType javaType = OBJECT_MAPPER.getTypeFactory().constructParametricType(ArrayList.class, clazz);
+        try {
+            return OBJECT_MAPPER.readValue(jsonString, javaType);
+        } catch (JsonProcessingException e) {
+            throw new IllegalStateException(DESERIALIZE_ERROR, e);
+        }
+    }
+
+    public static List<String> parseObjectsString(String jsonString){
+        JavaType javaType = OBJECT_MAPPER.getTypeFactory().constructParametricType(ArrayList.class, String.class);
+        try {
+            return OBJECT_MAPPER.readValue(jsonString, javaType);
+        } catch (JsonProcessingException e) {
+            throw new IllegalStateException(DESERIALIZE_ERROR, e);
+        }
+    }
+
+    public static String getString(JsonNode jsonNode, String fieldName){
+        if(jsonNode == null){
+            throw new NullPointerException();
+        }
+        if(jsonNode.get(fieldName) == null){
+            return null;
+        }
+        JsonNode result = jsonNode.get(fieldName);
+        if(result.isTextual()){
+            return jsonNode.get(fieldName).asText();
+        }
+        return jsonNode.get(fieldName).toString();
+    }
+
+    public static Integer getInteger(JsonNode jsonNode, String fieldName) {
+        if(jsonNode == null){
+            throw new NullPointerException();
+        }
+        if(jsonNode.get(fieldName) == null){
+            return null;
+        }
+        JsonNode value = jsonNode.get(fieldName);
+        if(value.isInt()){
+            return value.intValue();
+        }
+        throw new ClassCastException("can not cast to int, value : " + value);
+    }
+
+    public static Boolean getBoolean(JsonNode jsonNode, String fieldName){
+        if(jsonNode == null){
+            throw new NullPointerException();
+        }
+        if(jsonNode.get(fieldName) == null){
```

Comment:
We should create a local variable like this: JsonNode jsonNode = jsonNode.get(fieldName), because the jsonNode.get(fieldName) method is being called twice.